src/api/server.ts

```diff
@@ -0,0 +1,20 @@
+import express, { Express } from "express";
+import dotenv from "dotenv";
+import winston from "winston";
+import { health } from "./route/health";
+
+dotenv.config();
+
+export function runAPIServer(logger: winston.Logger): void {
+  const app: Express = express();
+  const port = process.env.PORT || 3000;
```

Comment:
Can we also please add `API_SERVER_HOST` env variable?
Setting the HOST to default `127.0.0.1` wouldn't work when running the bot within a Docker container.